src/centralized_telescope/setup.rs

```diff
@@ -0,0 +1,18 @@
+//! ALBA's Setup structure comprising, among others, the number of elements and internal parameters
+
+/// Setup output parameters
```

Comment:
Am I confused here, but I don't see the verifier threshold (the `n_f` in the paper)? Shouldn't it be part of the Setup struct?

Reply:
The verifier threshold (the n_f in the paper) is only needed for the setup parameter generations, hence it is not included in `Setup`